### DISCUSSION TITLE
What is the maximum number of static pages that can be generated with Nextra docs without increasing build time beyond 5 minutes?

### DISCUSSION THREAD

**Discussion by ryadav96** [shuding/nextra#1125] [Q&A]
@shuding 
I am encountering an error when trying to build my site with around 1500 dummy mdx pages. Is there a way to successfully build a site with a large number of pages without increasing the amount of memory used?

![image](https://user-images.githubusercontent.com/30411972/209481359-0b33ff83-aebe-49b8-a947-4c57a1172a9b.png)

**Comment by shuding** [comment]
Will track that in #1131 👍
